cluster/examples/kubernetes/ceph/cluster-test.yaml

```diff
@@ -0,0 +1,32 @@
+#################################################################################################################
```

Comment:
The issue I have with re-adding this file is that we would end up with more similar files. For instance, the "On PVC" use case and the other CRD CephFilesystem could have a `-test.yaml` too, from that perspective, it seems endless. Also, I can understand `cluster.yaml` might be underwhelming but in the end, it's well documented and it's a matter of reading it a bit.

Reply:
We already have the pattern of the `-test.yaml` suffix for CRs such as `filesystem-test.yaml` and `object-test.yaml`. It does provide a simple, quick install for the minikube case. While it is a developer convenience, it's also a convenience for others as mentioned in #5301. Personally, I'm tired of editing cluster.yaml every time I want to test something in minikube so I want the cluster-test.yaml back. :) I don't expect to need a -test.yaml for every example, it's just one per CR.

Reply:
I'm coming from #5301, having taken the same steps as @brightzheng100.

Putting `test-cluster.yaml` will be great, @travisn, but I feel that that documentation on storage in Minikube deployments should added. 

I agree that Quickstart doesn't seem like the right place, but [Bootstrapping Kubernetes](https://rook.io/docs/rook/master/k8s-pre-reqs.html) would be a great place. It says that "Rook will run wherever Kubernetes is running.", followed by a direction to start an default Minikube instance, which is a bit misleading.

It might also be a good place to [link](https://rook.io/docs/rook/v1.2/ceph-cluster-crd.html#cluster-settings) the `dataDirHostPath` changes `cluster.yaml` for data persistence.

### The Rabbit Hole
The `cephosd` storage error that @brightzheng100 mentions seems to be triggered by the ext4 filesystem on the Minikube VM's `/dev/vda1` drive.

The failure message 
`cephosd: skipping OSD configuration as no devices matched the storage settings for this node "minikube"`
is preceded by
`2020-04-22 22:03:02.849605 I | cephosd: skipping device "vda1" because it contains a filesystem "ext4"`
which reads the filesystem from `udevadm`
`2020-04-22 22:03:02.846556 D | exec: Running command: udevadm info --query=property /dev/vda1`

Ext4 could be whitelisted, even though [Ceph recommends against it.](https://docs.ceph.com/docs/jewel/rados/configuration/filesystem-recommendations/#not-recommended) this would make it easier to give Ceph a try, but I don't know if the object names will be short enough. The filesystem is probably configured in a `cephosd` daemon setting, but I didn't dive down to investigate it.

At that point, I was considering whether I needed to [manuallly zap](https://rook.io/docs/rook/v1.0/ceph-teardown.html) my Minikube VM, but instead I found #5301.